internal/services/order/req_res_body.go

```diff
@@ -1,18 +1,30 @@
 package order
 
+const Pending="PENDING"
```

Comment:
can be combine them like 
const (
//Statuses
Completed
Partial

//Order Types
Market
Limit
)

Reply:
done